docs/UserGuide.md

```diff
@@ -254,6 +261,8 @@ Format: `add_student n/NAME p/STUDENT_PHONE_NUMBER e/EMAIL a/ADDRESS l/STUDY_LEV
 :bulb: TutorBuddy does not restrict the <code>STUDY_LEVEL</code> due to the vast number of education levels. The user 
 is free to input any education level.<br>
 Example: Any of the inputs here will be accepted. <code>Sec 2</code>, <code>Secondary 2</code>, <code>JC 1</code>, <code>Poly Y2</code>
+:bulb: If you have different students of the same <code>NAME</code> consider changing the name. <br>
+Example: Add a <code>John Doe Bespectacled</code> and a <code>John Doe Unadorned</code>
```

Comment:
I think you can use a page break here.
Current implementation:
![image](https://user-images.githubusercontent.com/39006568/114258669-d2a78600-99fa-11eb-85c7-bb5a85488ba7.png)

add a <br> tag between line 263 and 264